src/Components/MovieDetails/MovieDetails.js

```diff
@@ -2,18 +2,29 @@ import React from 'react';
 import './MovieDetails.css'
 
 const MovieDetails = (props) => {
```

Comment:
why not destructure the props here?

Reply:
Just wanted to see if you would catch it =) JK, we will definitely destructure this. Looks like we overlooked it!